pilz_robot_programming/test/integrationtests/tst_api_cmd_conversion.py

```diff
@@ -358,6 +358,48 @@ def test_ptp_default_acc_scale(self):
         # +++++++++++++++++++++++
         self.assertEquals(EXP_VEL_SCALE*EXP_VEL_SCALE, req.max_acceleration_scaling_factor)
 
+    def test_ptp_global_scale(self):
+        """ Test the default acceleration scaling factor for ptp commands.
+
+            Test sequence:
+                1. Create ptp command.
+                2. Call convert function.
+                3. Set the scaling factor for the robot
+                4. Call the convert function.
+
```

Comment:
Also test creating a ptp command after setting the scaling factor?

Reply:
Y?

Reply:
Sorry, I mean *Call the convert function on a ptp command created after setting the scaling factor"
Imho this is should be tested.